intermediate_source/torchvision_tutorial.rst

```diff
@@ -304,6 +304,30 @@ be using Mask R-CNN:
 That’s it, this will make ``model`` be ready to be trained and evaluated
 on your custom dataset.
 
+Checking the model with random tensors (Optional)
+---------------------------
+
+Before iterating over the dataset, it's always good to see what the model 
+expects during training and inference time with random tensors. 
+
+.. code:: python
+
+   model = torchvision.models.detection.fasterrcnn_resnet50_fpn(pretrained=True) 
+   images,boxes,labels = torch.rand(4,3,600,1200), torch.rand(4,11,4), torch.rand(4,11) # For Training
+   images = list(image for image in images)                                             
+   targets = []                                   
+   for i in range(len(images)):
+       d = {}
+       d['boxes'] = boxes[i]
+       d['labels'] = labels[i].type(torch.int64)
```

Comment:
The use of `.type` is not encouraged (it's legacy), and instead we should use `.to`. See my fixes (including lint fixes) in https://github.com/pytorch/vision/pull/1713

Reply:
Thanks for pointing this out. I've fixed it.